docs/src/modules/java/pages/call-another-as-service.adoc

```diff
@@ -0,0 +1,118 @@
+= Calling other Akka Serverless services
```

Comment:
I think we want to have this in the JavaScript and Java specific pages instead, i.e. in the sdk repo.

Reply:
sorry, that is already the case